### PR TITLE
Major version upgrade of paragon

### PR DESCRIPTION
**Pull request by Mashal-m**
Major version upgrade of **@edx/paragon**

### CODE REVIEW

**Comment by muselesscreator** [comment]
I have no objection to the paragon update, but we don't need react-intl, as we have it imported through frontend-platform

File: package.json [hunk 1]
```diff
@@ -52,6 +52,7 @@
     "react": "16.14.0",
     "react-dom": "16.14.0",
     "react-helmet": "^6.1.0",
+    "react-intl": "^5.25.0",
```
Comment (muselesscreator) [inline package.json:55]:
```suggestion
```

Comment (Mashal-m) [reply package.json]: I have uninstalled it.

**Review by mattcarter — APPROVED** [review]
Approved

**Comment by adamstankiewicz** [comment]
@Mashal-m @muselesscreator Heads up that there was one additional step needed for this major upgrade to v20 of Paragon. Per the release notes, Paragon v20 now exposes a `paragonMessages` array containing the native translations for Paragon components but the `paragonMessages` are not yet passed to `@edx/frontend-platform` via the `initialize` function in this MFE. 

[inform] I created a GitHub issue about this follow-up task here: https://github.com/openedx/frontend-app-gradebook/issues/340

**Pull request merged**